face_api_dataset/dataset.py

```diff
@@ -140,6 +140,12 @@ class Modality(Enum):
     
     **Type**: `ndarray[float64]`. **Dimensions**: 2.
     """
+    FACE_BBOX = auto()
```

Comment:
We will need to regenerate the docs.

Reply:
How can I do that?

Reply:
I'll do this.

Reply:
`make html` in the docs directory will do the trick, but you'll need sphinx packages installed.